docs/src/main/sphinx/security/user-mapping.rst

```diff
@@ -95,9 +95,8 @@ also maps users in the form ``bob@uk.example.com`` to ``bob_uk``:
 .. literalinclude:: user-mapping.json
     :language: json
 
-Configure this example file to use the :doc:`LDAP </security/ldap>`
-authentication method (which uses the PASSWORD authentication type), as
-follows:
+In your coordinator's ``config.properties`` file, configure the following
+example to use the :doc:`LDAP </security/ldap>` PASSWORD authentication method:
 
```

Comment:
Someone led you astray in the original text, and it was probably me. The original line 98 has "Configure this example file." The "this" refers to the JSON file above, not anything below. So you're right that it's not correct to say we're "configuring" that file. Let's try "set up" with something like this:

> Set up the above example file to use the LDAP authentication method (which ... type) by adding the following line to your coordinator's ``config.properties`` file:

Remember that the only complaint about this file that caused this PR is that we were explicit about ``config.properties`` in **Pattern mapping rules** section, but then left readers to wonder what file to edit in the **File mapping rules** section.